digdag-core/src/main/java/io/digdag/core/agent/TaskCallbackApi.java

```diff
@@ -34,10 +34,30 @@ void retryTask(TaskRequest request, AgentId agentId,
     StoredSessionAttempt startSession(
             OperatorContext context,
             int siteId,
-            int projectId,
+            ProjectIdentifier projectIdentifier,
             String workflowName,
             Instant instant,
             Optional<String> retryAttemptName,
             Config overrideParams)
-        throws ResourceNotFoundException, ResourceLimitExceededException;
+            throws ResourceNotFoundException, ResourceLimitExceededException;
+
+    // Identifier of Project: id or name
+    class ProjectIdentifier
```

Comment:
Probably we can add `static` keyword to this

Reply:
In interface we don't need public and static keyword in my understand.
IntelliJ suggests as it and jls may also explain it.

https://stackoverflow.com/questions/2400828/inner-class-within-interface
https://docs.oracle.com/javase/specs/jls/se8/html/jls-9.html#jls-9.5

Reply:
Oh, sorry. I thought the file is for `class`.